app/file-detail/template.hbs

```diff
@@ -1,29 +1,161 @@
-{{title model.name}}
-<div class='row'>
-    <div class='col-sm-5'>
-        <h2>{{model.name}}</h2>
+{{title model.file.name}}
+{{quickfile-nav user=model.user onQuickfiles=false}}
+<div class='quickfiles-content row'>
+    <div class='col-sm-8'>
+        <h2>{{model.file.name}} <a id='versionLink' onclick={{action 'changeView'}}>{{concat "(Version: " mfrVersion ")"}}</a></h2>
```

Comment:
Minor: You don't need to use the helper concat here. Should be able to just do:
`>(Version: {{mfrVersion}})</a>`